components/brave_wallet_ui/components/asset-name-and-icon/index.tsx

```diff
@@ -25,7 +25,9 @@ export const AssetNameAndIcon = (props: Props) => {
   return (
     <StyledWrapper>
       <AssetIcon
-        src={assetLogo}
+        // This usage is good, but it's breaking because it looks like it's being loaded from chrome-untrusted://market-display frame
+        // the image is coming from https://assets.cgproxy.brave.com
+        src={`chrome://image?url=${encodeURIComponent(assetLogo)}&staticEncode=true16`} 
```

Comment:
last instance. This is for a separate issue that's occurring on `brave://wallet/crypto/explore/market`, but the behavior is the same. Trying to render a chrome://image URL from an untrusted frame